ChangeLog

```diff
@@ -1,3 +1,139 @@
+	  file evaluated when no file is specified with the '-f' option.
+	- Add an option to skip the initial policy run on bootstrap
+	  In some cases it may not be desired to run the policy as the last
+	  step of the bootstrap. This can be done with the new
+	  '--skip-bootstrap-policy-run' option for cf-agent. (CFE-2826)
+	- Trigger promises.cf as the last step of bootstrap (CFE-2826)
+	- Add support for overriding the package module's path (CFE-2103)
+	- Add support for setting package module interpreter (CFE-2880)
+	- Added --log-level option to all components
+	  This allows you to specify any log level (info, verbose, debug etc.).
+	  It is also less strict, allowing different spelling. As an example,
+	  --log-level i, --log-level INFO, --log-level inform are all the same.
+	- Added a new binary: cf-check
+	  Corrupt local databases (LMDB) continues to be a problem.
+	  cf-check will be used to diagnose and remediate problems
+	  with corrupt databases. It is a standalone binary, which
+	  doesn't evaluate policy or use the local databases, thus
+	  it can be used in situations where the other binaries
+	  like cf-agent would hang.
+	  cf-check replaces our lmdb database dumper, lmdump.
+	  `cf-check lmdump` or symlinking / renaming it to lmdump
+	  will make cf-check have the exact same behavior as lmdump.
+	  cf-check will include much more functionality in the future
+	  and some of the code will be added to other binaries,
+	  for example to do health checks of databases on startup.
+	  Ticket: (ENT-4064)
+	- Added function string_replace. (CFE-2850)
+	- Allow dots in variable identifiers with no such bundle
+	  As described and discussed in CFE-1915, defining remote variables
+	  (injecting variables into remote bundles) is dangerous and must
+	  be blocked. However, using a dot-separated common prefix for
+	  variables raises no security concerns and can be considered
+	  valid. (CFE-1915)
+	- Allow requiring TLS 1.3 as the minimum version
+	- Apply augments after vars, classes and inputs in def.json
+	  (CFE-2741, CFE-2844)
+	- Bundle name is now part of the log message when aborting a bundle
+	  (CFE-2793)
+	- Class names set by module protocol are automatically canonified
+	  (CFE-2877, CFE-2887)
+	- Classes failsafe_fallback and bootstrap_mode are now reported by default
+	- Correct log level for data_readstringarray* (CFE-2922)
+	- Do not iterate over JSON objects' properties in mustache (CFE-2125)
+	- Do not render templates when passed invalid data (CFE-2194)
+	- Eliminated error messages caused by attempting to kill expired processes
+	  (CFE-2824)
+	- Fix cf-runalerts systemd unit conditions so the service will run
+	  (ENT-3929)
+	- Fix the off-by-one error in cf-runagent background process spawning
+	  (CFE-2873)
+	- Fixed OOB read / heap buffer overflow in evaluator (ENT-4136)
+	- Fixed a memory leak which occured when reloading RSA keys from disk
+	- Fixed a memory leak which occured while loading augments files
```

Comment:
Did this RSA keys entry need a JIRA ticket reference?

Reply:
it came from this commit:
commit https://github.com/cfengine/core/commit/a03e82ba369b2fb3612a21d51d118aaad5417b74
Author: @olehermanse 
Date:   Tue Oct 30 14:28:33 2018 +0100

    Fixed a memory leak which occured when reloading RSA keys from disk
    
    Changelog: Title

from this PR: https://github.com/cfengine/core/pull/3396

So it should be CFE-2857, indeed